.github/workflows/release-new.yml

```diff
@@ -0,0 +1,104 @@
+# Copyright 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: release-new
+on:
+  workflow_dispatch:
+    inputs:
+      releaseType:
+        description: 'Release type: Major; Minor; Patch'
+        required: true
+permissions: read-all
+jobs:
+  NewRelease:
+    permissions:
+      contents: write
+      packages: write
+    runs-on: ubuntu-latest
+    env:
+      COSIGN_KEY_LOCATION: /tmp/cosign.key
+      COSIGN_PWD: ${{ secrets.COSIGN_PWD }}
+
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Docker Login
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Cosign Installer
+        uses: sigstore/cosign-installer@main
+        with:
+          cosign-release: 'v1.2.0'
```

Comment:
Can we remove patch here and get always the latest?

Reply:
don't think it's a good practice, one update could break our pipeline

Reply:
But will be a patch update. Why will break something? I'm saying use `v1.2` instead not `latest`

Reply:
I'm not sure why, but this action fails if passed a tag without patch version.

Reply:
at this case the release is a parameter to the action, if we want to version something would be the action [sigstore/cosign-installer@main](https://github.com/sigstore/cosign-installer)  or we could get the cosign [release](https://github.com/sigstore/cosign/releases) by the repository. When sign release step was built cosign was at 1.2.0 version, the one goreleaser used to do the integration, and the newer version from cosign(1.3.0) has some breaking changes that we don't want to deal with now.

Reply:
I'm just afraid we forgot to update security fixes, but ok.